static/app/gettingStartedDocs/ruby/rails.tsx

```diff
@@ -19,7 +19,8 @@ type Params = DocsParams;
 const getInstallSnippet = (
   params: Params
 ) => `${params.isProfilingSelected ? 'gem "stackprof"\n' : ''}gem "sentry-ruby"
-gem "sentry-rails"`;
+gem "sentry-rails"
+bundle install`;
```

Comment:
Not sure if I'm reading this right, but I think you have to separate `bundle install` into its own step. It's not a line you add to the Gemfile. It's a terminal command you run afterwards.

(Basically, it's like adding something to `package.json` then running `npm install` after.)